src/Components/NewNote/NewNote.jsx

```diff
@@ -27,11 +28,13 @@ const NewNote = () => {
     setNoteColor,
     showColor,
     setShowColor,
+    showLabel,
+    setShowLabel,
     editModal,
     setEditModal,
   } = usePebbleNote();
 
-  const { newInputTitle, showInput } = state;
+  const { newInputTitle, showInput, tempLabels } = state;
```

Comment:
You can use alias when destructing as in state is generally can have meaningful name